### ISSUE TITLE
NEXT COHORT — OG RN-20M wakes, new RN-100M, RN-300M enters Ruse School

### ISSUE THREAD

**Issue by bluegrouse92-lgtm** [N-A-pixel/Ruse-Public#11]
# NEXT COHORT — population expansion and first Ruse School-scale lineage

23 September 2026 marks a major population expansion after the first E3 cohort.

```text
OG RN-20M              awakened historical Generation-1 lineage
RN-8M-Q1               8,523,008 parameters
RN-8M-Q2               8,523,008 parameters
RN-8M-Q3               8,523,008 parameters
RN-8M-Q4               8,523,008 parameters
RN-100M-new            99,771,392 parameters
RN-300M-new-20260923  299,979,648 parameters
```

The quadruplet experiment is recorded in Issue #10. This issue records the wider cohort and the transition of Ruse School from architecture plan toward demonstrated research infrastructure.

## OG RN-20M

Authoritative tree:
`/home/deck/ruse_foundation_program/tokenizer_v1_c0_20260921/run_01`

Architecture:

```text
LlamaForCausalLM
layers:            10
hidden size:       384
attention heads:   6
KV heads:          2
FFN:               1024
context:           1024
vocabulary:        16,000
RoPE / RMSNorm / SwiGLU
tied embeddings
BF16
```

Authoritative preserved-artifact facts:

```text
exact parameters: 21,865,344
Day Zero SHA-256:
c47da781521c48b6640578063c05fc10d78593b8b120c2536fcb77e5546a54c8

protected C0 SHA-256:
cb2efe2c167f80c4d2e76477c79b195b6f4045925b3a6bd14330f3f5cca6fdbd

C0 age:             131,200 tokens
C0 steps:           513
C0 train loss:      4.9984
C0 validation loss: 5.3002
```

The intended continuation point is the protected C0 lineage, not Day Zero.

### Historical parameter-count correction

Early records state **21,880,704 parameters**. The preserved architecture/artifact audit gives **21,865,344**. The older number remains part of the historical record; future records use the audited exact count.

## RN-100M-new

A second independent ~100M-class native model now exists:

```text
parameters:        99,771,392
layers:            12
hidden size:       1024
attention heads:   16
KV heads:          4
FFN:               1408
context:           2048
vocabulary:        16,000
embeddings:        tied
seed:              2511141292
```

This provides a second independent observation at approximately the same scale rather than assuming parameter class uniquely determines developmental outcome.

## RN-300M-new-20260923

```text
model:       RN-300M-new-20260923
parameters:  299,979,648
tokenizer:   V1.1 lineage
```

Day Zero model SHA-256:
`77bc5787bb6087b860794fdb84c7eb4920e7d24687618ef5a472f75ef78eecc4`

The exact architecture is governed by its packaged `architecture_manifest.json`.

RN-300M deliberately extends the experiment to a scale intended for remote education on Oracle/Ruse School rather than occupying the Steam Deck with a long training run.

## Ruse School transport proof

RN-300M exercised the package/transfer/admission boundary:

```text
local package
→ Day Zero provenance binding
→ Oracle transfer
→ integrity/hash verification
→ return transfer
→ identity preserved
```

This demonstrated the transport/integrity boundary but did **not** yet prove substantial remote education end-to-end.

```text
Ruse School architecture planned      — demonstrated earlier
model package / transfer boundary     — exercised
remote integrity verification         — exercised
round-trip artifact preservation      — exercised
substantial remote education          — separate milestone
```

Compute location does not create a new model lineage; the provenance and education ledger preserve identity across the boundary.

## Population significance

The research population is now intentionally heterogeneous:

- tiny sibling models for controlled cohort/specialist research;
- the historical OG lineage resumed rather than discarded;
- independent models at similar parameter scales;
- established E3 lineages with known histories;
- Gary's separate 32K multilingual lineage;
- a 300M model intended for remote school-scale education.

**Status: OG AWAKENED. QUADRUPLETS BORN. SECOND ~100M BORN. 300M BORN. 300M RUSE SCHOOL TRANSPORT/INTEGRITY ROUND TRIP DEMONSTRATED.**

**Comment by bluegrouse92-lgtm** [comment]
## 24 September 2026 — RN-300M status clarification

RN-300M-new-20260923 completed E0 and its frozen evaluation. It is **not training now** and no RN-300M process is active on Oracle.

Canonical E0 checkpoint SHA-256:
`f273679d4c7cd4343d8e809f5fa4a83670114fde297b055a24b490c034889454`

E0 accounting:
- 2,022 / 2,022 sequences
- 258,816 optimized targets
- final training loss 5.0079
- validation: 9.8829 → 6.7951
- holdout: 9.9190 → 8.0967

Greedy probes still showed severe repetition, so the loss reduction is evidence of statistical learning, **not a claim of language competence or graduation**.
